apps/web/components/Organization/Wrapper/Themes/HljodbokasafnIslandsTheme/HljodbokasafnIslandsHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useMemo } from 'react'
+
+import { DefaultHeader, DefaultHeaderProps } from '@island.is/web/components'
+import { OrganizationPage } from '@island.is/web/graphql/schema'
+import { useLinkResolver, useNamespace } from '@island.is/web/hooks'
+
+import * as styles from './HljodbokasafnIslandsHeader.css'
+
+interface HeaderProps {
+  organizationPage: OrganizationPage
+}
+
+const HljodbokasafnIslandsHeader: React.FC<
+  React.PropsWithChildren<HeaderProps>
+> = ({ organizationPage }) => {
+  const { linkResolver } = useLinkResolver()
+  const namespace = useMemo(
+    () => JSON.parse(organizationPage.organization?.namespace?.fields ?? '{}'),
```

Comment:
Perhaps changing this from being a `??` to a `||` might be beneficial in case it's an empty string since `''` ?? `'{}'` will return `''`